orttraining/orttraining/test/python/orttraining_distributed_tests.py

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env python3
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+import sys
+import argparse
+
+from _test_commons import run_subprocess
+
+import logging
+
+logging.basicConfig(
+    format="%(asctime)s %(name)s [%(levelname)s] - %(message)s",
+    level=logging.DEBUG)
+log = logging.getLogger("Build")
+
+def parse_arguments():
+    parser = argparse.ArgumentParser()
+    parser.add_argument("--cwd", help="cwd")
+    return parser.parse_args()
+
+def main():
+    import torch
+    ngpus = torch.cuda.device_count()
+
+    if ngpus <= 1:
```

Comment:
I would raise an error if less than 2 is found, otherwise distributed train cant happen anyway in our context, right?

Reply:
Distributed train requires 2 or more gpus to run. Basically this returns 1 which would cause ```sys.exit``` to return 1 and cause a failure in the pipeline.
But let me raise an exception instead of returning 1.